app/models/organization.rb

```diff
@@ -80,6 +80,14 @@ def login_settings
     @login_settings ||= Mumukit::Auth::LoginSettings.new(login_methods)
   end
 
+  def absolute_link
+    "http://#{relative_link}"
```

Comment:
Hmm, this means we are explicilty removing https support. 

I don't know it is a big issue, but it may introduce bugs in the future